packages/react/src/format.test.tsx

```diff
@@ -138,13 +139,23 @@ describe("formatElements", function () {
     const elements = formatElements("<div><0/><0/></div>", {
       div: <div />,
       0: <span>hi</span>,
-    }) as Array<React.ReactElement>
+    })
+
+    expect(React.isValidElement(elements)).toBe(true)
+
+    assert.ok(
+      React.isValidElement<{
+        children: React.ReactElement[]
+      }>(elements)
+    )
+
+    const childElements = elements.props.children
+    const childKeys = childElements.map((el) => el?.key).filter(Boolean)
+
+    assert.ok(
```

Comment:
same as above please

Reply:
Similarly, this is done to satisfy the compiler, instead of an unsafe casting.
Up to you.

Reply:
Removed